qa/tasks/mgr/dashboard/test_rbd.py

```diff
@@ -510,3 +514,32 @@ def test_copy(self):
         self.assertStatus(204)
         self.remove_image('rbd_iscsi', 'coimg-copy')
         self.assertStatus(204)
+
+    def test_flatten(self):
+        self.create_snapshot('rbd', 'img1', 'snapf')
+        self._rbd_cmd(['snap', 'protect', 'rbd/img1@snapf'])
```

Comment:
I prefer to use our own REST API endpoints to do this, to get better coverage of our own code.
For instance, use: `self.update_snapshot('rbd', 'img1', 'snapf', None, True)`